### PR TITLE
AboutPage_Curtis

### PR DESCRIPTION
**Pull request by CurtisLam**
Re-adjusted footer display issue on web browser

### CODE REVIEW

File: src/screens/AboutScreen.tsx [hunk 1]
```diff
@@ -24,38 +21,61 @@ const AboutScreen = () => {
                 <VStack style={styles.banner}>
                     <Heading size="3xl">Welcome to BucketList!</Heading>
                     <Text fontSize="16">
-                        Check off and complete your wish list with friends, and
-                        keep all your memory without regret.
+                        <em>
+                            Check off and complete your wish list with friends,
+                            and keep all your memory without regret.
+                        </em>
```
Comment (bscuron) [inline src/screens/AboutScreen.tsx]: <em> tag is not a react native tag

File: src/screens/AboutScreen.tsx [hunk 2]
```diff
@@ -24,38 +21,61 @@ const AboutScreen = () => {
...
+                            interactive list of activities you may want to
+                            accomplish. With this list, you can connect with old
+                            and new friends to complete these activities and
+                            live a more fulfilled life. Many times, we want to
+                            do things but never do them. BucketList is intended
+                            to get people out and to accomplish their goals.
+                            With the help of social connection and common
+                            interests, users can connect and live the life they
+                            want to live instead of daydreaming but never doing.
+                        </em>
```
Comment (bscuron) [inline src/screens/AboutScreen.tsx]: <em> tag is not a react native tag

File: src/screens/AboutScreen.tsx [hunk 3]
```diff
@@ -24,38 +21,61 @@ const AboutScreen = () => {
...
-                         your friends upcoming events too! 
-                        We hope you can check off your list and try fun activities you might not have thought about!
+                        <em>
+                            Through social connection and event tracking and
+                            planning we can all reach our goals. Not only will
+                            your upcoming events be available on your bucket
+                            list, you can also view your friends upcoming events
+                            too! We hope you can check off your list and try fun
+                            activities you might not have thought about!
+                        </em>
```
Comment (bscuron) [inline src/screens/AboutScreen.tsx]: <em> tag is not a react native tag

File: src/screens/AboutScreen.tsx [hunk 4]
```diff
@@ -24,38 +21,61 @@ const AboutScreen = () => {
...
-                
             </VStack>
             <Spacer size="10" />
-            <Container position="fixed" bottom="0" alignSelf="center">
-                <FooterView />
+            <Spacer size="10" />
+            <Container alignSelf="center">
+                <video width="320" height="240" controls>
+                    <source src="./BucketList_intro.MP4" type="video/mp4" />
+                </video>
```
Comment (bscuron) [inline src/screens/AboutScreen.tsx]: <video> and <source> are not react native tags

File: src/screens/AboutScreen.tsx [hunk 5]
```diff
@@ -24,38 +21,61 @@ const AboutScreen = () => {
...
 export default AboutScreen;
 
+const Video = () => {
+    return (
+        <video width="320" height="240" controls>
+            <source src="BucketList_intro.mp4" type="video/mp4" />
+        </video>
+    );
+};
+
```
Comment (bscuron) [inline src/screens/AboutScreen.tsx]: Invalid tags used

**Review by bscuron — CHANGES_REQUESTED** [review]
Please see changes

**Review by CurtisLam — COMMENTED** [review]
Removed imcompatible tag from about screen

**Review by bscuron — APPROVED** [review]

**Pull request merged**